accounts/services/database/__init__.py

```diff
@@ -27,3 +31,46 @@ def get_session(id: int) -> Optional[TapirSession]:
         return None
     except SQLAlchemyError as e:
         raise IOError('Database error: %s' % e) from e
+
+def create_session(user_data: UserData) -> SessionData:
+    """
+    Create a new legacy session.
+
+    Parameters
+    ----------
+    user_data : :class:`.UserData`
+
+    Returns
+    -------
+    :class:`.SessionData`
+    """
+
+    tapir_session: TapirSession = TapirSession(
+        user_id = user_data.user_id,
+        last_reissue = int(user_data.last_reissue),
+        start_time = int(user_data.start_time),
+        end_time = int(user_data.end_time)
+    )
+
+    data = json.dumps({
```

Comment:
We should think about what we actually need to return for the cookie. Perhaps we only need the session ID?

Reply:
probably, I'll leave it in for now and remove it later once I start looking at cookies in detail

Reply:
As per discussion, I'll go ahead and remove this for now